tests/spec/s-array.js

```diff
@@ -1568,4 +1568,301 @@ describe('Array', function () {
             expect(result).toEqual(['2', '3']);
         });
     });
+
+    describe('#sort()', function () {
+        var descending = function (left, right) {
+            var leftS = String(left),
+                rightS = String(right),
+                val;
+            if (leftS === rightS) {
+                val = +0;
+            } else if (leftS < rightS) {
+                val = 1;
+            } else {
+                val = -1;
+            }
+            return val;
+        };
+        var testArr1 = [];
+        for (var i = 0; i < 100; i += 1) {
+            testArr1.push(('00' + (Math.random() * 100).toFixed(0)).slice(-3));
```

Comment:
please use `beforeEach` for setup like this - also it'd be preferable to confine that to a "describe" for only the tests that need it.